components/SearchResults/index.js

```diff
@@ -1,50 +1,79 @@
 import { useStore } from "../../src/store/store";
-import { useEffect, useState } from "react";
+import { Fragment, useEffect, useState } from "react";
 import { cards } from "../../lib/data";
 import TopMenuBar from "../Styled/StyledTopMenuBar";
 import StyledNavbar from "../Styled/StyledNavbar";
 import styled from "styled-components";
 import Link from "next/link";
-
-const ListContainer = styled.div`
+import Image from "next/image";
+import CardPreviewImage from "../CardPreviewImage";
+const ListContainer = styled.ul`
   display: flex;
   flex-direction: column;
-  background: orange;
-  height: 30em;
-  overflow: hidden;
+  background: black;
+  height: 18em;
+  overflow: auto;
+  color: white;
+  list-style-type: none;
+  margin: 0;
+  padding-left: 36px;
+  padding-right: 36px;
+  font-size: 1.2em;
 `;
 
-const StyledList = styled.ul`
+const StyledList = styled.div`
```

Comment:
All fine, just remember that naming a div as a "StyledList" doesn't follow 100% semantic HTML rules for SEO and accessibility ;) But it still is possible though.